.config/dotnet-tools.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "fake-cli": {
+      "version": "5.19.0",
+      "commands": [
+        "fake"
+      ]
+    },
+    "paket": {
+      "version": "5.226.0",
```

Comment:
and used the opportunity to change paket approach to the prefered one  https://fsprojects.github.io/Paket/get-started.html#NET-Core-preferred